src/components/BaseWarningBlock.vue

```diff
@@ -5,6 +5,9 @@ defineProps({
   },
   externalHref: {
     type: String
+  },
+  linkText: {
```

Comment:
Hmm, if warning block and info block are essentially the same, why not make it a 

BaseMessageBox 
with a 

level: "info" | "warning" | "error" prop